### PR TITLE
Add a banner for Summer '22 Meetups

### PR DESCRIPTION
**Pull request by exitflynn**
:sunglasses:

### CODE REVIEW

**Comment by exitflynn** [comment]
wait one min i'll try ta SQUASH them commits into one

**Comment by osBins** [comment]
Image looks great!
1. I hope the elements used in the image are freely-usable
2. Can the size be reduced to about 50-75%? Whatever looks good to you.

**Comment by exitflynn** [comment]
The image was made using Canva and I have Canva Pro. That gives me what Canva calls a Pro License for the image which gives _me_ full permission to use it for non-commercial/commercial resources, so it is "freely-usable" for ME and i decide to use it in a non-commercial manner, so it's freely usable for all now! 
[(link)](https://www.canva.com/en_in/help/licenses-copyright-legal-commercial-use/)
tl;dr: yes freely usable fr fr very much

2. Will give it a shot

**Comment by exitflynn** [comment]
1. The image was made using Canva and I have Canva Pro. That gives me what Canva calls a Pro License for the image which gives _me_ full permission to use it for non-commercial/commercial resources, so it is "freely-usable" for ME and i decide to use it in a non-commercial manner, so it's freely usable for all now! [(link)](https://www.canva.com/en_in/help/licenses-copyright-legal-commercial-use/) 
tl;dr: yes freely usable fr fr very much, I'm a really good lawyer

2. Will give it a shot :+1:

**Comment by exitflynn** [comment]
how does this look to you?

**Comment by osBins** [comment]
Amazing! Merging your PR.
